### ISSUE TITLE
FP: array_merge in loop

### ISSUE THREAD

**Issue by kalessil** [kalessil/phpinspectionsea#1348]
```php
foreach ($results as $key => $value) {
    $results[$key]['breakdown'] = \array_merge($results[$key]['breakdown'], $extraAccessorials);
}
```

**Comment by akrz** [comment]
Here is a complete example of the false positive

```php
$results = [
	['id' => 1, 'breakdown' => [1, 2]],
	['id' => 2, 'breakdown' => [5, 6]],
	['id' => 3, 'breakdown' => [7]],
];
 
$extraAccessorials = [9];
 
foreach ($results as $key => $value) {
    $results[$key]['breakdown'] = \array_merge($results[$key]['breakdown'], $extraAccessorials);
}
```

**Comment by kalessil** [comment]
Fixed!
